packages/synapse-react-client/esbuild.config.mjs

```diff
@@ -50,7 +50,7 @@ const esBuildOptions = {
       namedExport: 'ReactComponent',
       exportType: 'named',
     }),
-    ESBuildNodePolyfillsPlugin,
+    NodeModulesPolyfillPlugin(),
```

Comment:
Any issues with these React components in SWC after this change?

Reply:
Using Chrome local overrides, I found that this likely breaks the SWC build.  So thanks for asking!